react-native-web peer dependency

## Summary

`react-native-maps` has a peer dependency conflict with the latest `expo init` package.

```
npm ERR! Found: react-native-web@0.13.18
npm ERR! node_modules/react-native-web
npm ERR!   react-native-web@"~0.13.12" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react-native-web@"^0.11" from react-native-maps@0.28.0
npm ERR! node_modules/react-native-maps
npm ERR!   react-native-maps@"^0.28.0" from the root project
```

Is there any issue with updating the `react-native-web`?

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
   "peerDependencies": {
     "react": ">= 16.0 || < 18.0",
     "react-native": ">= 0.51",
-    "react-native-web": "^0.11"
+    "react-native-web": "^0.13"
   },
   "devDependencies": {
     "@react-native-community/eslint-config": "0.0.5",

